lib/metadata_json_deps/runner.rb

```diff
@@ -53,6 +63,14 @@ def run
     rescue Interrupt
     end
 
+    def check_deprecated(version)
+      if version.to_s.eql?('999.999.999') || version.to_s.eql?('99.99.99')
```

Comment:
No need for an if/else block here. Just the boolean expression on its own is all you need as it will be implicitly returned.